### ISSUE TITLE
Cannot save per-core or per-game shader presets

### ISSUE THREAD

**Issue by psy-q** [flathub/org.libretro.RetroArch#65]
When I try to save a shader preset for a core, I get an error in the interface ("Error saving shaders") and the following on the console:

```
Failed to create directory: "/app/share/libretro/shaders/presets/"
```

Changing the config option to:

```
video_shader_dir = "~/.var/app/org.libretro.RetroArch/share/shaders"
```

Fixes it.

**Comment by RobLoach** [comment]
Will this one do it? https://github.com/flathub/org.libretro.RetroArch/pull/67

**Comment by psy-q** [comment]
I will get Flatpak Builder tonight and try to build the flatpak, but from the looks of it I'm already 99% sure it's fixed like that.

**Comment by psy-q** [comment]
I'm unable to build it (`Error: module nvidia-cg-toolkit: Invalid buildsystem: "simple"`) but I'm trusting that this fixes the issue :)

**Comment by jahed** [comment]
Hi, this issue isn't solved yet as #67 was not merged. Is there a known workaround?
